api/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+class MovieSerializer(serializers.Serializer):
+
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100, required=True)
+    description = serializers.CharField(max_length=200)
+    duration = serializers.IntegerField(required=True)
+
+    def create(self, validated_data):
```

Comment:
Warto dodawać typowanie na takich metodach. To w odniesieniu do wszystkich
```suggestion
    def create(self, validated_data: dict) -> Movie:
```